Wordle.Api/Wordle.Api/Identity/IdentitySeed.cs

```diff
@@ -0,0 +1,57 @@
+﻿﻿using Microsoft.AspNetCore.Identity;
+using Wordle.Api.Data;
+
+namespace Wordle.Api.Identity;
+
+public static class IdentitySeed
+{
+    public static async Task SeedAsync(UserManager<AppUser> userManager, RoleManager<IdentityRole> roleManager)
+    {
+        await SeedRolesAsync(roleManager);
+
+        await SeedAdminAsync(userManager);
+    }
+
+    private static async Task SeedRolesAsync(RoleManager<IdentityRole> roleManager)
+    {
+        if (!await roleManager.RoleExistsAsync(Roles.Admin))
+        {
+            await roleManager.CreateAsync(new IdentityRole(Roles.Admin));
+        }
+    }
+
+    private static async Task SeedAdminAsync(UserManager<AppUser> userManager)
```

Comment:
Hmm, I don't seem to be able to log in with either of these accounts
![image](https://github.com/IntelliTect-Samples/EWU-CSCD379-2023-Spring/assets/43832670/457b8645-085e-4cff-9ede-4e57bde4cb99)